Behavior tree switch

vytasrgl opened the pull request.

linas commented:
OK, I will review this more carefully tomorrow, but I don't like if for two reasons. Fisrst, for starting and stopping the bot, that can be done more easily and better in python or scheme or whatever. So creating this `(DefinedPredicate "Is tree paused?")` is pointless and serves no purpose. Just do it in  python; its simpler that way.

By contrast, this python code: `exp.name = random.choice(["happy","comprehending","engaged"])` is EXACTLY what I want to avoid.  It places state and control in a location where it is impossible to access and alter.  The whole point of converting the core to opencog is that the states and the control become accessible to the learning and to the reasoning and memory subsystems, so that the robot can know why it did and why it did it, and talk about it. If it's buried in python code, it can't do that -- opencog is much much too dumb to read, understand and modify python code.  The goal is that, eventually, significant parts of the behavior tree are things that the robot learned on its own, rather than having you or me or Wenwei or ManHin or David write explicit code.  So we are writing explicit code, for now, for the short-term, to bootstrap.    So, on/off things, like pausing the robot, should almost certainly NOT be done in opencog; its pointless and serves no purpose.  But expression control has to be opencog, cause that is what makes the state accessible to language and learning.

vytasrgl commented:
I modified start/stop procedure. Also removed the expression sending as it was really not used in chatbot still, so we will wait for OC to handle chatbot to add some expression to responses.

linas commented on src/put_atoms.py:
```diff
@@ -45,3 +45,10 @@ def chatbot_affect_happy(self):
+	# manages the behavior trees state
+	def btree_stop(self):
+		scheme_eval_h(self.atomspace, "(halt)")
+
```
yes, exactly!

linas commented:
sorry for delay; I'm looking at the other issues too, e.g. the affect stuff
